client/my-sites/controller.js

```diff
@@ -111,9 +111,9 @@ function renderNoVisibleSites( context ) {
 	);
 }
 
-function isPathAllowedForDomainOnlySite( pathname ) {
+function isPathAllowedForDomainOnlySite( pathname, domainName ) {
 	const urlPrefixesWhiteListForDomainOnlySite = [
-		'/domains/manage/',
+		`/domains/manage/${ domainName }`,
```

Comment:
Should we use [this path builder](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/paths.js#L11)?